### ISSUE TITLE
Docker image latest/0.3.1 - panicked at 'there is no timer running...'

### ISSUE THREAD

**Issue by dylanholmes** [jasonwhite/rudolfs#28]
In attempting to run rudolfs server via the image uploaded to docker I'm getting the following error:

```
[+] Running 1/0
 ⠿ Container rodulfs-demo_app_1  Recreated                                                                                          0.0s
Attaching to app_1
app_1  |  2021-06-03T18:43:12.188Z INFO  rudolfs > Initializing storage...
app_1  |  2021-06-03T18:43:12.188Z INFO  rudolfs::storage::s3 > Connecting to S3 bucket '<!--redacted-->' at region '<!--redacted-->'
app_1  | thread 'main' panicked at 'there is no timer running, must be called from the context of a Tokio 0.2.x runtime', /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/tokio-0.2.25/src/time/driver/handle.rs:24:32
app_1  | note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
app_1 exited with code 101
```

---
Here is what I did:

```
# Go to a working directory
mkdir rodulfs-demo
cd rodulfs-demo

# Get the default docker compose configuration
curl -O https://raw.githubusercontent.com/jasonwhite/rudolfs/master/docker-compose.yml

# Export environment variables required by rudolfs
export AWS_ACCESS_KEY_ID=<...>
export AWS_SECRET_ACCESS_KEY=<...>
export AWS_DEFAULT_REGION=<region of S3 bucket>
export LFS_ENCRYPTION_KEY=<Paste key saved to Secrets Manager>
export LFS_S3_BUCKET=<S3 bucket for LFS storage>
export LFS_MAX_CACHE_SIZE=10GiB

docker compose up
```

--- 
I also tried switching to the 0.3.0 docker image and got the same error. I will try the 0.2.x versions next, but I see I'll need to change my docker-compose.yml file first.

**Comment by jasonwhite** [comment]
Yikes. It looks like the `backoff` crate needs to be updated. That's used for retries. This bug probably wasn't caught because retries don't often happen in testing. :/

I'll take a look at fixing this later today.

**Comment by dylanholmes** [comment]
Awesome! Thanks for the quick response too.

Btw, I tried using 0.2.11, and no issues there.

**Comment by jasonwhite** [comment]
Should be fixed in v0.3.2. Just waiting on CI to publish the new Docker image.

**Comment by dylanholmes** [comment]
Thanks!
